ports/esp32/machine_sdcard.c

```diff
@@ -57,6 +57,10 @@
 // Forward declaration
 const mp_obj_type_t machine_sdcard_type;
 
+//fix the sdspi highspeed select error
+esp_err_t sdmmc_init_card_hs_mode_fix(sdmmc_card_t* card);
+esp_err_t sdmmc_card_init_fix(const sdmmc_host_t* config, sdmmc_card_t* card);
```

Comment:
What are these for?

Reply:
I think i want to try using more higher freq for the sdmmc module at that time which idf lib limited, but failed. Already remove that code.